salt/modules/rh_ip.py

```diff
@@ -692,7 +655,7 @@ def _parse_settings_eth(opts, iface_type, enabled, iface):
             for opt in vlan:
                 result[opt] = opts[opt]
 
-    if iface_type not in ["bond", "vlan", "bridge", "ipip"]:
+    if iface_type not in ("bond", "team", "teamport", "vlan", "bridge", "ipip"):
```

Comment:
I believe you want to only include "team" here.

The team interface is the one, that does not get a hwaddr assigned as it automatically gets them from the team member ports.

Teamport needs a HWADDR entry. Also compare https://github.com/saltstack/salt/pull/57773 which fixes a logic error here.